contrib/spec/podman.spec.in

```diff
@@ -34,8 +34,8 @@
 
 # People want conmon packaged with the copr rpm
 %global import_path_conmon      github.com/containers/conmon
-%global git_conmon      https://%{import_path_conmon}
-%global commit_conmon   8455ce1ef385120deb827d0f0588c04357bad4c4
+%global git_conmon      https://github.com/haircommander/conmon-exec
```

Comment:
Can we point this to containers/conmon now? Or are we still waiting on upstream merges?

Reply:
Waiting on https://github.com/cri-o/cri-o/pull/2377, then we need to set conmon to build from upstream cri-o instead of RPM, then we need to get it into c/conmon, then we can get it here. Still a long road ahead :)

Reply:
the second step is on my list to do concurrently with getting this in, but things keep popping up.